[release/7.0][wasm] Fix Wasm.Build.Tests with stable 7.0.0

radical opened the pull request:
`Wasm.Build.Tests` started to fail when installing the workload manifests with https://github.com/dotnet/runtime/pull/76849 which moved to `Stable branding for GA`. This caused the workload installation to fail because the targets were using the incorrect manifest version - `7.0.0-ci` instead of `7.0.0`. It is fixed here by setting that early, and using that in all the relevant places.
This revealed a second issue - https://github.com/dotnet/sdk/issues/28607 - which caused tests to break, and will be fixed by https://github.com/dotnet/sdk/pull/28627 .

Fixes https://github.com/dotnet/runtime/issues/76908 .

## Customer Impact

No impact, since this is a *tests-only* change.

## Testing

Running `Wasm.Build.Tests`.

## Risk

No risk, since this is a *tests-only* change.

radical committed:
[wasm] Fix Wasm.Build.Tests with stable 7.0.0

Set the manifest package version to use (`7.0.0` instead of `7.0.0-ci`)
early.

Fixes https://github.com/dotnet/runtime/issues/76908 .

radical committed:
workloads-testing.targets: ignore symbols.nupkgs when looking for runtime packs

radical committed:
Fix WBT to have the correct version for WORKLOAD_PACKS_VER

carlossanlop commented:
Whenever you get a chance, can you please take a look at the CI failures? LMK when this is good to merge.

radical commented:
Working on that, I'll ping you when this is ready.

radical committed:
[wasm] Emit an error if PackageVersionForWorkloadManifests property is not set

radical committed:
workloads-testing: Get the PackageVersion at build time

.. because it is not set at evaluation time. So, get it in a target, and
move the WorkloadIds to a target too.

radical committed:
Ensure the version is set before generating the runscript for WBT

radical committed:
[wams] Update Mono.Toolchain's WorkloadManifest.targets to not depend on

.. the import order.

radical committed:
revert workloadManifest.target changes

radical commented:
The Wasm.Build.Tests failures will be fixed by https://github.com/dotnet/sdk/pull/28627 . Once, an installer with that change is available, this should be green, and can be merged.

carlossanlop commented:
There are failures showing up in the wasm legs. Are any related to this change?

carlossanlop commented:
This is blocking the runtime branding PR as described here: https://github.com/dotnet/runtime/pull/77549#issuecomment-1297660028

radical committed:
[wasm] Disable failing WBT tests due to https://github.com/dotnet/runtime/issues/77740

radical committed:
[wasm] Run WBT against 7.0.1xx builds

radical committed:
[wasm] Disable more WBT failing tests

radical committed:
Disable more blazor relinking tests

radical commented:
@carlossanlop this is good to merge. The WasmBuildTests seem to be stuck on SIMD tests, but those are not affected by this at all.

radical commented:
And once this is merged, then we can merge upstream into https://github.com/dotnet/runtime/pull/77549 .
